components/server/test/ome/server/itests/pixeldata/PersistentEventLogLoaderTest.java

```diff
@@ -273,9 +274,9 @@ protected Pixels pixels() {
         pixels.setSizeZ(1);
         pixels.setSizeT(1);
         pixels.setSizeC(1);
-        pixels.setSha1("UNKNOWN");
-        pixels.setPixelsType(new PixelsType("int8"));
-        pixels.setDimensionOrder(new DimensionOrder("XYZCT"));
+        pixels.setSha1(ChecksumAlgorithm.VALUE_ADLER_32);
```

Comment:
it is, I assume that it was similar to "pending..." we have in various locations
Happy to revert if we want to test the insert of such value